Jamulus.pro

```diff
@@ -157,7 +157,11 @@ win32 {
         MACOSX_BUNDLE_ICON.path = Contents/Resources
         QMAKE_BUNDLE_DATA += MACOSX_BUNDLE_ICON
     } else {
-        QMAKE_INFO_PLIST = mac/Info-make.plist
+        equals(QT_VERSION, "5.9.9") {
```

Comment:
The only thing I can ask about this change is whether an explicit `equals 5.9.9` here is correct.  Does this apply "before 5.10", for example?

Reply:
5.9.9 is the version of Qt we use for a legacy build. There is a qmake function for comparing version numbers, but it only became available in 5.10, so wouldn't work here. I thought the simplest solution was just to compare with the actual version number we specify in the autobuild workflow, as it's for the benefit of the CI, not for people's own builds.

Reply:
Assuming "5.9.9" appears only here in the Jamulus.pro file, that's probably okay, really.

We should really get the auto build running for _all_ targets in the Jamulus.pro file to make sure what it claims to offer works.  I notice there's a "jackonmac" target.  Should we also build that and does it need both legacy and non-legacy builds?

Reply:
> Assuming "5.9.9" appears only here in the Jamulus.pro file, that's probably okay, really.

It only appears here and in `.github/workflows/autobuild.yml`. So long as they match, it should be fine.

> We should really get the auto build running for _all_ targets in the Jamulus.pro file to make sure what it claims to offer works. I notice there's a "jackonmac" target. Should we also build that and does it need both legacy and non-legacy builds?

I haven't seen a demand for jackonmac. I guess the very few that might want it would build it themselves.
From `git log --patch Jamulus.pro`, it looks like Volker added it in 2016, for version 3.4.2. It is probably years since it was last tested. But that's out of scope for this PR.

Reply:
Yep, I don't like stale code though, so it triggers an urge to get stuff tested...

Reply:
The lowest Qt version yet.
I build Jamulus with Qt 5.6.1 on IA32 CentOS 6.
Thus you could specify Qt 5.6.1 instead of Qt 5.9.9 (unless MacOS can't).

> > Assuming "5.9.9" appears only here in the Jamulus.pro file, that's probably okay, really.
> 
> It only appears here and in `.github/workflows/autobuild.yml`. So long as they match, it should be fine.